Sources/MapboxMobileEvents/CLLocation+MMEMobileEvents.m

```diff
@@ -25,11 +25,19 @@ - (CLLocationAccuracy)mme_roundedVerticalAccuracy {
 }
 
 - (CLLocationAccuracy)mme_roundedSpeedAccuracy {
```

Comment:
Probably not a related question to this PR, but I couldn't find the usage of this method in the SDK 🤔  
Who is gonna call those functions? Do they expect to get `-1.0`?